packages/manager/apps/web/client/app/email-domain/upgrade/upgrade.html

```diff
@@ -0,0 +1,55 @@
+<div class="page-header mb-5">
+    <h1 data-ng-bind="$ctrl.serviceName"></h1>
+</div>
+
+<div data-ovh-alert="{{ $ctrl.alerts.main }}"></div>
+
+<div class="container-fluid px-5">
+    <oui-back-button data-sref="^">
```

Comment:
```suggestion
    <oui-back-button data-href="^">
```
the `sref` binding doesn't seem to exist https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/back-button
Moreover, I think it would be better to inject it through routing